background.js

```
@@ -0,0 +1,443 @@
+  const patternRegex = pattern
+    .replace(/\./g, '\\.')
+    .replace(/\*/g, '.*');
+  return new RegExp(`^${patternRegex}$`).test(url);
+}
+
+/**
+ * Gets the domain from a URL
+ * @param {string} url - The URL to get the domain from
+ * @returns {string} The domain
+ */
+function getDomain(url) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getDomain'. Consider to delete it or rename it.

'getDomain' is defined here:


(lint/suspicious/noRedeclare)